app/assets/javascripts/angular/controllers/series/index.js.erb

```diff
@@ -0,0 +1,8 @@
+app.controller('SeriesIndexCtrl', ['$scope', '$http', '$location', function($scope, $http, $location){
+
+  $http.get("<%= ENV['URL'] %>" + '/series').success(function(response){
```

Comment:
if you have a `localhost:3000/` and you add `/series`... you are going to get a double slash: `localhost:3000//series' ...